Services/RelationService.cs

```diff
@@ -0,0 +1,179 @@
+        }
+
+        public void Create(RelationViewModel relation)
+        {
+            if (relation != null)
+            {
+                var relationToCreate = CreateRelationFromRelationViewModel(relation);
+                relationRepo.Create(relationToCreate);
+            }
+        }
+
+        public void Update(RelationViewModel relation)
+        {
+            if (relation != null)
+            {
+                var relationToUpdate = CreateRelationFromRelationViewModel(relation);
+                relationRepo.Update(relationToUpdate);
+            }
+        }
+
+        public void Delete(params Guid[] identificators)
+        {
+            relationRepo.Delete(identificators);
+        }
+
+        public List<RelationViewModel> GetSortedAndFilteredRelations(IQueryable<Relation> list, string categoryName, string orderProperties)
+        {
+            if (!string.IsNullOrEmpty(categoryName))
+            {
+                //Filters collection of Relations by choosed category
+                list = GetRelationsByCategories(list, orderProperties);
+            }
+
+            if (!string.IsNullOrEmpty(orderProperties))
+            {
+                //ordering collection by choosed properties
+                list = GetSortedRelationsByProperties(list, orderProperties);
+            }
+
+            return ViewModelMapper.Map<List<Relation>, List<RelationViewModel>>(list.ToList());
+        }
+
+        //TODO write an algorithm for applying postal mask
+        public string ApplyPostalCodeMask(string postalCode, string postalCodeFormat)
+        {
+            return postalCode;
+        }
+
+        public IMapper ConfigureMapperForDto()
+        {
+            var mapper = new MapperConfiguration(cfg => cfg.CreateMap<Relation, RelationDTO>()
+               .ForMember("Street", opt => opt.MapFrom(c => c.RelationAddress.Street))
+               .ForMember("City", opt => opt.MapFrom(c => c.RelationAddress.City))
+               .ForMember("Country", opt => opt.MapFrom(c => c.RelationAddress.CountryName))
+               .ForMember("PostalCode", opt => opt.MapFrom(c => c.RelationAddress.PostalCode))
+               .ForMember("StreetNumber", opt => opt.MapFrom(c => c.RelationAddress.Number))).CreateMapper();
+
+            return mapper;
+        }
+
+        public IMapper ConfigureMapperForViewModel()
+        {
+            var mapper = new MapperConfiguration(cfg => cfg.CreateMap<Relation, RelationViewModel>()
+               .ForMember("Street", opt => opt.MapFrom(c => c.RelationAddress.Street))
+               .ForMember("City", opt => opt.MapFrom(c => c.RelationAddress.City))
+               .ForMember("Country", opt => opt.MapFrom(c => c.RelationAddress.CountryName))
+               .ForMember("PostalCode", opt => opt.MapFrom(c => c.RelationAddress.PostalCode))
+               .ForMember("StreetNumber", opt => opt.MapFrom(c => c.RelationAddress.Number))).CreateMapper();
+
+            return mapper;
```

Comment:
It is not a part of this service. You should move it to a separate class. Remember "S" in solid.